drivers/include/shtcx.h

```diff
@@ -99,10 +101,10 @@ int8_t shtc1_read(const shtc1_t *dev, uint16_t *rel_humidity, int16_t *temperatu
  * @param[in] dev       The I2C device descriptor.
  * @param[in] id        ID of the device.
  *
- * @return              SHTC1_OK on everything done.
- * @return              SHTC1_ERROR_BUS on error.
+ * @return              SHTCX_OK on everything done.
```

Comment:
Please add a line break in `@details, the line is longer than 100 characters. Feel free to squash it directly.